docs/sources/dashboards/create-reports/_index.md

```diff
@@ -331,19 +318,29 @@ To edit a report, follow these steps:
 1. When you've finished making changes, click **Confirm** at the top of the screen to go to the last step.
 1. Click **Update report**.
 
-### Pause or resume a report
+### Pause or resume reports {#pause-a-report}
 
-You can pause and resume sending reports from the report list view. To do this, follow these steps:
+You can pause and resume sending reports from the report list view.
+To do this, follow these steps:
 
 1. In the main menu, click **Dashboards > Reporting**.
 1. On the row of the report you want to update, do one of the following:
 
    - Click the pause icon - The report won't be sent according to its schedule until it's resumed.
    - Click the resume icon - The report resumes on its previous schedule.
 
+### Delete reports
+
+To delete a report, follow these steps:
+
+1. In the main menu, click **Dashboards > Reporting**.
+1. On the row of the report you want to update, click the trash can icon.
+1. Click **Delete** to confirm.
+
 ## Troubleshoot Reporting
 
-To troubleshoot and get more log information, enable debug logging in the configuration file. Refer to [Configuration](ref:configuration) for more information.
+To troubleshoot and get more log information, enable debug logging in the configuration file.
+Refer to the [log filters configuration documentation](ref:log-filters) for more information.
 
 ```bash
 [log]
```

Comment:
I can't write a suggestion because you didn't update it but the config should now be:
```
filters = rendering:debug,report.api:debug,report.render:debug,report.scheduler:debug,report.sender:debug,report.service:debug
```
```